Extension/ui/settings.html

```diff
@@ -538,18 +538,18 @@
             <option value="windows-msvc-x64">windows-msvc-x64</option>
             <option value="windows-msvc-arm">windows-msvc-arm</option>
             <option value="windows-msvc-arm64">windows-msvc-arm64</option>
-            <option value="clang-x86">clang-x86</option>
-            <option value="clang-x64">clang-x64</option>
-            <option value="clang-arm">clang-arm</option>
-            <option value="clang-arm64">clang-arm64</option>
-            <option value="gcc-x86">gcc-x86</option>
-            <option value="gcc-x64">gcc-x64</option>
-            <option value="gcc-arm">gcc-arm</option>
-            <option value="gcc-arm64">gcc-arm64</option>
-            <option value="msvc-x86">msvc-x86</option>
-            <option value="msvc-x64">msvc-x64</option>
-            <option value="msvc-arm">msvc-arm</option>
-            <option value="msvc-arm64">msvc-arm64</option>
+            <option value="clang-x86">clang-x86 (legacy)</option>

```

Comment:
Would "deprecated" be more understandable or is that not the correct term to use?

Reply:
"deprecated" usually means that something will be removed in the future. "legacy" means something is old be is being kept for backward compatibility.